### DISCUSSION TITLE
put rewards in f2p minigames

### DISCUSSION THREAD

**Discussion by Ironman12495** [runelite/runelite#18120] [Ideas]
i know there is rewards in castle wars but,put in clan wars or something

**Comment by Felanbird** [comment]
RuneLite is a client for playing the game, we don't make in-game changes. You'll have to make a Reddit/Twitter post with your suggestion like everyone else.

**Reply by Ironman12495** [reply]
ok

**Comment by YvesW** [comment]
We're not jagex, we don't make the game. We maintain the third-party RuneLite client.

**Reply by Ironman12495** [reply]
ok
